src/main/java/com/example/survey/models/SurveyController.java

```diff
@@ -13,7 +13,6 @@ public SurveyController(SurveyRepo surveyRepo) {
         this.surveyRepo = surveyRepo;
     }
 
-    @CrossOrigin(origins = "http://localhost:3000")
```

Comment:
So it is the same origin if it has a different port?

Reply:
This isn't needed. I forgot to remove it in my initial PR when I was trying to get CORS requests working. 

Instead, we specify a [`proxy`](https://github.com/vikrambombhi/SimpleSurvey/blob/master/client/package.json#L14) in the react app and the react development server will handle the requests for us appropriately.